docs/en/integrations/data-ingestion/kafka/index.md

```diff
@@ -856,6 +855,106 @@ Sink, use [Kafka Connect Transformations](https://docs.confluent.io/platform/cur
+| `connector.class` | `"com.clickhouse.kafka.connect.ClickHouseSinkConnector"` | Connector Class(set and keep as the default) |
+| `tasks.max` | `"1"` | The number of Connector Tasks |
+| `errors.retry.timeout` |`"60"` | ClickHouse JDBC Retry Timeout |
+| `exactlyOnce` | `"false"` | Exactly Once Enabled |
+| `topics` | `""` | The Kafka topics to poll - topic names must match table names |
+| `value.converter` | `"org.apache.kafka.connect.json.JsonConverter"` | Connector Value Converter |
+| `value.converter.schemas.enable` | `"false"` | Connector Value Converter Schema Support |
+| `errors.tolerance` | `"none"` | Connector Error Tolerance |
+| `errors.deadletterqueue.topic.name` | `""` | If set, a DLQ will be used for failed batches |
+| `errors.deadletterqueue.context.headers.enable` | `""` | Adds additional headers for the DLQ |
+| `clickhouseSettings` | `""` |  Allows configuration of ClickHouse settings, using a comma seperated list (e.g. "insert_quorum=2, etc...") |
+
+### Configuration Recipes
+These are some common configuration recipes to get you started quickly.
+#### Basic Configuration
+The most basic configuration to get you started - it assumes you're running Kafka Connect in distributed mode and have a ClickHouse server running on `localhost:8443` with SSL enabled, data is in schemaless JSON.
+```json
+{
+  "name": "clickhouse-connect",
+  "config": {
+    "connector.class": "com.clickhouse.kafka.connect.ClickHouseSinkConnector",
+    "tasks.max": "1",
+    "database": "default",
+    "errors.retry.timeout": "60",
+    "exactlyOnce": "false",
+    "hostname": "localhost",
+    "port": "8443",
+    "ssl": "true",
+    "username": "default",
+    "password": "<PASSWORD>",
+    "topics": "<TOPIC_NAME>",
+    "value.converter": "org.apache.kafka.connect.json.JsonConverter",
+    "value.converter.schemas.enable": "false",
+    "clickhouseSettings": ""
+  }
+}
+```
+
+#### Basic Configuration with Multiple Topics
+The connector can consume data from multiple topics
+```json
+{
+  "name": "clickhouse-connect",
+  "config": {
+    "connector.class": "com.clickhouse.kafka.connect.ClickHouseSinkConnector",
+    ...
+    "topics": "SAMPLE_TOPIC, ANOTHER_TOPIC, YET_ANOTHER_TOPIC",
+    ...
+  }
+}
+```
+
+#### Basic Configuration with DLQ
+```json
+{
+  "name": "clickhouse-connect",
+  "config": {
+    "connector.class": "com.clickhouse.kafka.connect.ClickHouseSinkConnector",
+    ...
+    "errors.deadletterqueue.topic.name": "<DLQ_TOPIC>",
+    "errors.deadletterqueue.context.headers.enable": "true",
+  }
+}
+```
+#### Using with different data formats
+##### Avro Schema Support
+It assumes Schema Registry is available on `http://localhost:8081`
+```json
+{
+  "name": "clickhouse-connect",
+  "config": {
+    "connector.class": "com.clickhouse.kafka.connect.ClickHouseSinkConnector",
+    ...
+    "value.converter": "io.confluent.connect.avro.AvroConverter",
+    "value.converter.schema.registry.url": "http://localhost:8081",
```

Comment:
"http://localhost:8081" -> "http://{hostname}:8081"

Reply:
That's why the comment says `It assumes Schema Registry is available on `http://localhost:8081`